lib/logster/redis_store.rb

```diff
@@ -422,7 +426,12 @@ def grouping_key
 
     def register_rate_limit(severities, limit, duration, callback)
       severities = [severities] unless severities.is_a?(Array)
-      rate_limiter = RedisRateLimiter.new(@redis, severities, limit, duration, callback)
+      redis = (@redis_raw_connection && @redis_prefix) ? @redis_raw_connection : @redis
```

Comment:
@SamSaffron This feels a little awkward for me. The user has to pass in both `@redis_raw_connection` and `@redis_prefix` in order for `@redis_raw_connection` to be used. Any idea how I can make the config better?